tests/guest-tools/unix/test_guest_tools_unix.py

```diff
@@ -34,7 +33,8 @@ def _check_os_info(self, vm, vm_distro):
         detected_distro = vm.distro()
         assert detected_distro == vm_distro
 
-    def test_install(self, running_vm, state):
+    @pytest.fixture(scope="class", autouse=True)
+    def vm_install(self, running_vm, state):
```

Comment:
I don't like it that this tests is made a fixture, but for the the sake of removing `@pytest.mark.incremental`, I'll pretend I haven't seen it.

Reply:
Yep, I don't see a better option either.